src/Controllers/ReinstateProductController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Controllers;
+
```

Comment:
Not super important, but the default is 2 lines of whitespace either side of namespace line I believe